agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -119,6 +119,17 @@ bool slave_thread::init()
         return false;
     }
 
+    if (config.ucc_listener_slave_hostap_iface == config.hostap_iface &&
```

Comment:
Please also add a flag which disables UCC completely in both agent and controller since it has effect on the code.

Reply:
There is already. It is the port number. If the port is zero, it means disable. (look one line down).

Reply:
So this should be mentioned in a comment in the configuration file if it isn't already.

Reply:
done